layouts/shortcodes/lang.html

```diff
@@ -15,6 +16,15 @@
 
         <div class="language-prologue-typescript"></div>
         <div>{{ $inner }}</div>
+    {{ else if eq . "dotnet" }}
+        <div class="language-prologue-csharp"></div>
+        <div>{{ $inner }}</div>
+
+        <div class="language-prologue-fsharp"></div>
+        <div>{{ $inner }}</div>
+
+        <div class="language-prologue-visualbasic"></div>
+        <div>{{ $inner }}</div>
```

Comment:
@justinvp does this work if we didn't actually write an example for any language other than c#?

Reply:
Yeah, it'll work. It's some extra content that's generated on the page that isn't strictly necessary since we're only showing the C# tab, but it won't be visible to anyone.